src/pages/learn.js

```diff
@@ -211,14 +240,37 @@ export default ({
                     <FeaturedArticles articles={featuredArticles} />
                 </HeaderContent>
             </Header>
-            <Article>
+
+            <TabBar
+                handleClick={setActiveItem}
+                leftTabs={leftTabs}
+                rightTabs={rightTabs}
+                activeItem={activeItem}
+            />
+
+            {(isLoadingVideos || isLoadingPodcasts) && <P>Loading...</P>}
+
+            <CardContainer>
                 <StyledFilterBar
                     filters={filters}
                     filterValue={filterValue}
                     setFilterValue={updateFilter}
                 />
-                <CardList articles={articles} />
-            </Article>
+                {activeItem === 'All' && (
+                    <CardList
+                        articles={articles}
+                        videos={videos}
+                        podcasts={podcasts}
+                    />
+                )}
+                {activeItem === 'Articles' && <CardList articles={articles} />}
+                {activeItem === 'Videos' && <CardList videos={videos} />}
+                {activeItem === 'Podcasts' && <CardList podcasts={podcasts} />}
+            </CardContainer>
+
+            {(errorVideos || errorPodcasts) && (
```

Comment:
Similar thing here, even if these fail we still have articles. Another Dan question for later but let's omit for now since *some* content exists.